docs/gatsby-node.js

```diff
@@ -58,6 +50,36 @@ exports.onCreateNode = async ({ node, actions, getNode }) => {
   }
 }
 
+exports.createSchemaCustomization = ({ actions, schema }) => {
```

Comment:
This creates a `Contributor` type (that defaults to `[]`) and adds it to the `Mdx.fields`. 

We do this because:
- it's an optional field meant only for `guides` collection pages
- we currently have no nodes with `fields.contributors` since all the guides are on `dev-ts`, so gatsby fails when it sees a query for `contributors` as that type isn't defined